source/clear-linux/tutorials/dlrs/dlrs.rst

```diff
@@ -3,81 +3,95 @@
 Deep Learning Reference Stack
 #############################
 
-This tutorial shows you how to run benchmarking workloads in |CL-ATTR| using
-TensorFlow\* or PyTorch\* with the Deep Learning Reference Stack. We also
-cover using Kubeflow for multi-node benchmarking.
+This tutorial describes how to run benchmarking workloads for TensorFlow\*,
+PyTorch\*, and Kubeflow in |CL-ATTR| using the Deep Learning Reference Stack.
+
 
 .. contents::
    :local:
    :depth: 1
 
-The Deep Learning Reference Stack is available in five versions:
+Overview
+********
+
+We created the Deep Learning Reference Stack to help AI developers deliver the
+best experience on Intel® Architecture. This stack reduces complexity common
+with deep learning software components, provides flexibility for customized
+solutions, and enables you to quickly prototype and deploy Deep Learning
+workloads. Use this tutorial to run benchmarking workloads on your solution.
 
-* `Intel MKL-DNN-VNNI`_, which is optimized using Intel® Math Kernel Library for Deep Neural Networks (Intel® MKL-DNN) primitives and introduces support for AVX-512 Vector Neural Network Instructions (VNNI).
```

Comment:
Open Q about name change from VNNI to DL Boost captured in email.

Reply:
I'll work with the engineering team on this -- it's the actual name of the docker image on DockerHub, so will need to be updated there first and then have links updated.